### ISSUE TITLE
Can't type letter 'r' into search bar

### ISSUE THREAD

**Issue by EvHaus** [facebook/react-devtools#1156]
This is super bizarre, but I can't seem to type the letter `r` into the search bar:

![2018-09-14 16 34 05](https://user-images.githubusercontent.com/226640/45579385-0c608200-b83c-11e8-8dff-3c680ed06349.gif)

In the gif I'm trying to type "orgtax" and "tree", but I end up with "ogtax" and "tee".

Copy/pasting works, but typing into the input doesn't work. It's as if there's code that says:

```js
onKeyDown (event) {
   if (event.keyCode === 'r') return;
}
```

I'm running React Developer Tools 3.3.2 on Chrome 71.0.3552.2

**Comment by ngyikp** [comment]
It's a Chrome dev bug: https://bugs.chromium.org/p/chromium/issues/detail?id=882422

**Comment by bvaughn** [comment]
Interesting!

FWIW I'm unable to repro this with the same DevTools version and Chrome 67.0.3396.87 (OSX).

**Comment by ngyikp** [comment]
Fixed on 71.0.3557.0 canary

**Comment by bvaughn** [comment]
Thanks @ngyikp! Since this doesn't seem like a React DevTools bug, I'm going to close this issue.

**Comment by EvHaus** [comment]
> Fixed on 71.0.3557.0 canary

Confirmed. Thanks! All good now.
